api/config/cdr_config_preprod.json

```diff
@@ -70,15 +70,15 @@
     },
     {
       "cdrVersionId": 5,
-      "name": "All of Us Controlled Tier Dataset v5 alpha 1",
+      "name": "All of Us Controlled Tier Dataset v5 alpha 2",
       "isDefault": true,
       "bigqueryProject": "fc-aou-cdr-preprod-ct",
       "bigqueryDataset": "C2021Q2R1",
       "wgsBigqueryDataset": "alpha1_1000",
```

Comment:
FYI: I will be updating this probably tomorrow for next week's release, were you planning to let this go out with the release? Or push out of band?

Reply:
I was going to do an out of band publish, that way I can do a bit of validation on the CDR indices build. We made a few changes since the last build and I want to confirm that things looks right.

Reply:
Done.